src/cpp/channel.h

```diff
@@ -27,11 +27,9 @@ class Channel : public GCObject<ChannelData> {
 
     size_t size();
 
-public:
-    Channel() = delete;
-
 private:
-    explicit Channel(const sol::stack_object& capacity);
+    Channel() = default;
+    void initialize(const sol::stack_object& capacity);
```

Comment:
It would be nice to add comment why we need separate function initialization.

Reply:
There is no good place for such comment, so I'll add it to GC